linux/ca-certificates/debian/src/main/packaging/Dockerfile

```diff
@@ -1,4 +1,6 @@
-FROM debian:bullseye
+ARG CONTAINER_REGISTRY=""
+
+FROM ${CONTAINER_REGISTRY}debian:bullseye
```

Comment:
Before we had this FROM clause, what registry did we pull from?  I'm not sure we had that hard coded to bullseye as a default, although I do see it listed as stable below

Reply:
There was no registry set so we were pulling from the default one that is `docker.io`. We had it hardcoded to bullseye by default.